Week3/homework/js-exercises/areAmazing.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+{
+  // Exercise 1: You are amazing, Noer!
+
+  // Write a function named giveCompliment
+  // function giveCompliment() { };
+
+  // It takes 1 argument: your name
+  // function giveCompliment(yourname) { };
+
+  // Inside the function create an array with 10 strings.
+  // Each string should be a compliment, like "great", "awesome"
+
+  // let yourname = '';
+
+  function giveCompliment(yourname) {
+    const myArr = ['great', 'awesome', 'corking', 'magnificent', 'brave', 'nice', 'cool', 'fantastic', 'beautiful', 'wonderful'];
+
+    // Write logic that randomly selects a compliment
+    let compliment = myArr[Math.floor(Math.random() * myArr.length)];
```

Comment:
1. `myArr` isn't the best name for this variable. Maybe `complimentsList`?
2. `const` instead of `let` for `compliment` variable please.